Lesson31/Merve/src/shared-components/typography/BodyM/styles.css

```diff
@@ -0,0 +1,5 @@
+.bodyM {
+    font-size: 24px;
```

Comment:
Font-size should actually be 12px.
<img width="380" alt="Screenshot 2024-09-23 at 14 10 56" src="https://github.com/user-attachments/assets/6141a7be-cf20-4c76-b520-ffda8c43affb">